app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Idv
+  module HybridMobile
+    module Socure
+      class DocumentCaptureController < ApplicationController
+        include Idv::AvailabilityConcern
+        include DocumentCaptureConcern
+        include Idv::HybridMobile::HybridMobileConcern
+
+        before_action :check_valid_document_capture_session
+
+        def show
+          Funnel::DocAuth::RegisterStep.new(document_capture_user.id, sp_session[:issuer]).
+            call('hybrid_mobile_socure_document_capture', :view, true)
+
+          # document request
+          document_request = DocAuth::Socure::Requests::DocumentRequest.new(
+            document_capture_session_uuid: document_capture_session_uuid,
+            redirect_url: idv_hybrid_mobile_socure_document_capture_url,
+            language: I18n.locale,
```

Comment:
the Socure Document Request documentation specifies simplified chinese as `zh-cn` ... providing our code `zh` is unexpected per the spec and renders the CaptureApp in English